src/hooks/useTheme.ts

```diff
@@ -0,0 +1,7 @@
+import { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+
+export const useTheme = () => {
+  const themeContext = useContext(ThemeContext);
+  return themeContext;
```

Comment:
p5;
`themeContext` 자체를 재사용하실 목적이 아니시라면 아래처럼 한 줄로 return해도 될 거 같습니다!
```suggestion
  return useContext(ThemeContext);
```